src/appmixer/google/analytics/CreateEvents/component.json

```diff
@@ -1,134 +1,118 @@
 {
-    "name": "appmixer.google.analytics.TrackCheckout",
-    "author": "Ian Sietro <ian@client.io>",
-    "description": "Send a Checkout Tracking hit to your Google Analytics view (profile).",
+    "name": "appmixer.google.analytics.CreateEvents",
+    "author": "Zbynek Pelunek <zbynek.pelunek@client.io>",
+    "description": "Creates an event with user interaction data.",
     "auth": {
-        "service": "appmixer:google",
-        "scope": [
-            "https://www.googleapis.com/auth/analytics",
-            "https://www.googleapis.com/auth/analytics.readonly"
-        ]
-    },
-    "quota": {
-        "manager": "appmixer:google:analytics",
-        "maxWait": 5000,
-        "resources": "requests",
-        "scope": {
-            "userId": "{{userId}}"
-        }
+        "service": "appmixer:google:analytics"
```

Comment:
is the authentication needed? there is no authenticated call in the implementation